.travis.yml

```diff
@@ -38,3 +39,12 @@ matrix:
 
       script:
         - make test
+
+    - language: java
+
+      script:
+        - (cd conode/; make docker)
```

Comment:
why do we need `docker` and `docker_dev`?

Reply:
we don't need `make docker`, but I added it there just to make sure it builds. maybe it belongs to somewhere else instead of the java test. should I moved it to the go tests?

Reply:
go tests look better to me.